code.user.js

```diff
@@ -1115,6 +1130,21 @@
                             totalPriceWithoutFeesOnMarket += task.sellPrice;
                             totalPriceWithFeesOnMarket += market.getPriceIncludingFees(task.sellPrice);
                             updateTotals();
+                        } else if (data != null && isRetryMessage(data.message)) {
+                            logDOM(padLeft +
+                                ' - ' +
+                                itemName +
+                                ' retrying listing because ' +
+                                data.message
```

Comment:
First char should be lowercase (see example below).

Reply:
Fixed